qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -477,27 +492,24 @@ def __init__(
                 waveform to 1. The default is ``True`` and the amplitude is constrained to 1.
 
         """
-        import sympy as sym
+        parameters = {"amp": amp if isinstance(amp, complex) else complex(amp), "sigma": sigma}
```

Comment:
I don't think this works if amp is a `ParameterExpression`.

Reply:
Thanks. That's my mistake. This must be `amp if isinstance(amp, ParameterExpression) else complex(amp)`. Added a test for this.